clients/pkg/promtail/targets/docker/target.go

```diff
@@ -95,7 +95,7 @@ func (t *Target) processLoop(ctx context.Context) {
 		Timestamps: true,
 		Since:      strconv.FormatInt(t.since, 10),
 	}
-
+	inspectInfo, err := t.client.ContainerInspect(ctx, t.containerName)
```

Comment:
You need to handle the error :slightly_smiling_face: 
```suggestion
	inspectInfo, err := t.client.ContainerInspect(ctx, t.containerName)
	if err != nil {
        level.Error(t.logger).Log("msg", "could not inspect container info", "container", t.containerName, "err", err)
		t.err = err
        return
	}
```